hw-6/microsoft/css/style.scss

```diff
@@ -0,0 +1,340 @@
+/* Переменные */
+
+$grey: #262626;
+$background: #F3F3F3;
+
+/* Шаблоны */
+
+%padding {
+    padding: 0 64px;  
+}
+
+%nav-link {
+    margin: 0 10px;
+    text-decoration: none;
+    color: $grey;
+    font-size: 13px;
+}
+
+%nav-link-icons {
+    width: 13px;
+    height: 13px;
+    padding: 10px;
+    text-decoration: none;
+    color: $grey;
+    font-size: 13px;
+}
+
+%nav-link-hover {
+    border-bottom: 2px solid $grey;
+    padding-bottom: 3px;
+}
+
+%footer-link {
+    font-size: 11px;
+    line-height: 16px;
+    color: #616161;
+    text-decoration: none;
+}
+
+/* Миксины */
+
```

Comment:
move to top of file

Reply:
Mixiny idut pered kodom, v kotorom oni  ispolzuyutsa